word_scripts/word_file_merger.py

```diff
@@ -0,0 +1,19 @@
+
+path_to_save = '../app/src/main/assets/words/'
+file_prefix = 'words-length-'
+
+def word_merger(dict_of_words):
+    for length, words in dict_of_words.items():
+        with open('{0}{1}{2}'.format(path_to_save, file_prefix, length), 'r') as file:
+            wordsByLength = file.readlines()
+        
+        for word in words:
+            if '{0}\n'.format(word) in wordsByLength:
+                print(word + " already in list")
+            else:
+                wordsByLength.append('{0}\n'.format(word))        
+
+        with open('{0}{1}{2}'.format(path_to_save, file_prefix, length), 'w') as file:
+            for word in wordsByLength:
+                file.write(word)
+                            
```

Comment:
Can you figure out a way to just append to the file instead of rewriting the whole file?